### PR TITLE
front: fix open data train import

### LINKED ISSUE #4210: Rolling stock selection not working in open data train import
### What happened?

When selecting a rolling stock in the open data train import module, the selector remains empty.

image

### What did you expect to happen?

Selecting a rolling stock should work and should display the selected one.

### How can we reproduce it (as minimally and precisely as possible)?

1. Go to any scenario
2. Try to select a rolling stock

### What operating system, browser and environment are you using?

- **Browser**: Chromium
- **OS**: Ubuntu
- **Env**: Dev

### OSRD version (top right corner `Account` button > `Informations`)

4c97ff14

### LINKED ISSUE #4211: Steps not displayed in open data train import
### What happened?

In a scenario, after importing open data trains and selecting "complete pathfinding" (not automatically), the step to place on the map is undefined instead showing the name of the step.

image

### What did you expect to happen?

The name of the step should be displayed

image

### How can we reproduce it (as minimally and precisely as possible)?

1. Go to a scenario
2. Import open data trains
3. Try to generate de path manually (first button)

### What operating system, browser and environment are you using?

- **Browser**: Chromium
- **OS**: Ubuntu
- **Env**: Dev

### OSRD version (top right corner `Account` button > `Informations`)

4c97ff14

### LINKED ISSUE #4212: front: migrate RTK endpoint in ImportTrainScheduleModal.tsx
### Description and goal

Use RTK for endpoints calls

### Acceptance criteria

The endpoints are all called with rtk

### PR DESCRIPTION
**Pull request by SharglutDev**
Close #4210 
Close #4211 
Close #4212

### CODE REVIEW

File: front/public/locales/fr/operationalStudies/importTrainSchedule.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
     "errorNoFrom": "Vous devez renseigner une origine.",
     "errorNoTo": "Vous devez renseigner une destination.",
     "errorSameFromTo": "L'origine et la destination doivent être différentes.",
-    "unableToRetrievePathfinding": "Impossible de créer un chemin"
+    "unableToRetrievePathfinding": "Impossible de créer un chemin",
+    "unableToRetrieveTrainSchedule": "Impossible de créer les calcul de marches"
```
Comment (clarani) [inline front/public/locales/fr/operationalStudies/importTrainSchedule.json]: I suggest this :) 
```suggestion
    "unableToRetrieveTrainSchedule": "Impossible de réaliser les calculs de marche"
```

File: front/src/applications/operationalStudies/components/ImportTrainSchedule/ImportTrainScheduleConfig.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React, { useState, useContext } from 'react';
 import { useTranslation } from 'react-i18next';
-import RollingStockSelector from 'common/RollingStockSelector/RollingStockSelector';
+import RollingStockSelector from 'common/RollingStockSelector/WithRollingStockSelector';
```
Comment (clarani) [inline front/src/applications/operationalStudies/components/ImportTrainSchedule/ImportTrainScheduleConfig.tsx:3]: Nice !!! Thank you

**Review by clarani — CHANGES_REQUESTED** [review]
Can not test (because error in the backend), but LGTM ! :)

File: front/public/locales/fr/operationalStudies/importTrainSchedule.json (hunk 1 above)
Comment (SharglutDev) [reply front/public/locales/fr/operationalStudies/importTrainSchedule.json]: Done

**Review by clarani — APPROVED** [review]
LGTM !

**Pull request merged**